addons/im_livechat/models/discuss_channel.py

```diff
@@ -434,6 +434,21 @@ def _email_livechat_transcript(self, email):
             'body_html': mail_body,
         })
         mail.send()
+        if (
+            not self.env.user._is_public()
```

Comment:
I think this condition is not needed. Public partners won't ever be in `customer_partner_ids`.